media-plugins/dragonfly-reverb/dragonfly-reverb-9999.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A set of free reverb effects"
+HOMEPAGE="https://github.com/michaelwillis/dragonfly-reverb"
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3 autotools
+	EGIT_REPO_URI="https://github.com/michaelwillis/dragonfly-reverb.git"
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/michaelwillis/dragonfly-reverb/releases/download/${PV}/DragonflyReverb-Source-v${PV}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/DragonflyReverb-Source-v${PV}"
+fi
+LICENSE="GPL-3"
+SLOT="0"
+IUSE="jack"
+
+RDEPEND="x11-libs/libX11
+virtual/opengl
+jack? ( virtual/jack )
+"
+
+DOCS=( README.md )
+
+src_prepare() {
+	default
+
+	# Remove automagic deps
+	sed -i -re '/^HAVE_(CAIRO|OPENGL|X11|JACK|LIBLO)/d' dpf/Makefile.base.mk || die 'sed failed'
+	# Do not strip
+	sed -i -re 's/-Wl,--strip-all//g' common/Makefile.mk
+}
+
+src_configure() {
+	echo "Nothing to configure"
+}
+
+src_compile() {
```

Comment:
Since this is using DPF, does this build all plugins formats (DSSI, LV2, VST)?
I added a bunch of build options for Nekobi (another DPF based plugin) to allow the choice of what to build, see here https://github.com/simonvanderveldt/Nekobi/commits/master
Not sure if we want this level of control/granularity or not? AFAIK DSSI requires liblo so if we can't control which plugin types are built we might need to add that as a dep (assuming it's a build-time dep).

Reply:
This is an effect, so I guess you mean LADSPA, not DSSI. But as far as I can tell, this plugin always unconditionally builds LV2 and VST versions, and can optionally build standalone JACK versions.

I already get rid of HAVE_LIBLO, so even if you have it installed it won't try to use it. But even if I set it to true it won't build LADSPA versions anyway, so I think all of that is not used by DragonFly. I don't think there are any extra deps for the LV2/VST builds, so we might as well build them unconditionally.